src/integration-test/java/com/commercetools/sync/integration/externalsource/products/ProductSyncWithAssetsIT.java

```diff
@@ -288,7 +285,7 @@ public void sync_withMatchingProductWithDuplicateAssets_shouldFailToUpdateProduc
             executeBlocking(productSync.sync(singletonList(productDraft)));
 
         // Assert results of sync
-        assertThat(syncStatistics).hasValues(1, 0, 1, 0);
+        assertThat(syncStatistics).hasValues(1, 0, 0, 0);
```

Comment:
Before, there was always an update happening, since we were always issuing a setPrice action to copy the prices. But now after there is a price sync. We don't do that anymore, we only update if needed based on built actions from comparison.